### ISSUE TITLE
Change text "...package files install completed. Performing other installation steps."

### ISSUE THREAD

**Issue by DarwinJS** [chocolatey/choco#1090]
The text "...package files install completed." seems confusing because the install then goes on to download the installer and actually install the software.

I'm not sure what is complete at that point, but it would be helpful to use terminology that does not confuse it with the actual software install or just remove the message altogether.

Thanks!

**Comment by ferventcoder** [comment]
Chocolatey has published documentationbthat clearly delineates "package" from "software" - those words are there to help folks understand that the packaging files are in place and stemmed out of conversations where it was confusing that packaging was upgraded vs when software was upgraded. Since the automation script calls are not required for Chocolatey, we needed something there. 

We are definitely open to suggestions on how to better word this.

**Comment by DarwinJS** [comment]
How about "...Chocolatey package files download and staged.  Ready to begin software installation."

Or any variation of that avoids the term 'install' with regard to the mentioned files being in a complete state and prefaces "package files" with "Chocolatey" to disambiguate them from the software installation package.

I think it is helpful for onboarding people to the platform if the chosen terminology can be crafted to avoid ambiguities without the need to consult documentation for platform specific terminology.

Now if you're building packages, not just consuming them, then you're gonna be reading a lot and chocolatey's documentation is essential for that activity.

**Comment by ferventcoder** [comment]
I agree with everything above, however I think the wording needs a little more thought. Right now "Ready to begin software installation" shoehorns only one use of Chocolatey packages (albeit a pretty big one - installer packages).

Run `choco install baretail`. There is a runtime binary, no software installation. No archive to unzip. The packaging files being added is literally the "install".

**Comment by ferventcoder** [comment]
The messaging should be succinct and sufficient to work with all scenarios.

**Comment by ferventcoder** [comment]
Apologies that I was not very clear with 

> Since the automation script calls are not required for Chocolatey, we needed something there.

**Comment by DarwinJS** [comment]
By "software installation" I was meaning "the binaries you want delivered to your box will be attempted now" - regardless of whether a setup.exe, zip or straight copy was required to do so.

But may be a more generic "Ready to begin package processing." ?  

Or just leave out the second phrase all together?

Either of the above two options handles my concern adequately.

**Comment by ferventcoder** [comment]
> regardless of whether a setup.exe, zip or straight copy was required to do so.

Here:
![image](https://cloud.githubusercontent.com/assets/63502/21269308/d8a9f1d6-c377-11e6-93e5-21b317793d21.png)

Since this is a very valid scenario and folks with internal packages sometimes have only this, this should be met as well.

**Comment by ferventcoder** [comment]
The "package files install" is all that the package does.

**Comment by DarwinJS** [comment]
I see what you are saying.  The install is complete at that point.

I would probably still separate the messages.  So have only  "...Chocolatey package files download and staged." for all package types.

And ONLY for packages that are only file copies and are essentially complete, add an additional message "Installation Complete."

That might work for both circumstances even though in the case of copying EXEs they are the same operation under the covers?
